### PR TITLE
Currency Conversion: Adds price as a keyword

### LINKED ISSUE #3129: Currency Conversion: Trigger more for bitcoin queries
I don't know if it should be this IA or one of the others, but there's a bunch of queries where people are looking for the price of bitcoin and we're not triggering:

[https://duckduckgo.com/?q=bitcoin+price](https://duckduckgo.com/?q=bitcoin+price)
[https://duckduckgo.com/?q=bitcoin+value](https://duckduckgo.com/?q=bitcoin+value)
[https://duckduckgo.com/?q=price+of+bitcoin](https://duckduckgo.com/?q=price+of+bitcoin)

User Feedback:
_"These searches should yield the price up top without additionally clicking on a link to find it."_

------
IA Page:  http://duck.co/ia/view/currency 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @MrChrisW

### PR DESCRIPTION
**Pull request by adityatandon007**
## Related Issues and Discussions

Fixes #3129 

## People to notify
@moollaza  @pjhampton 

Instant Answer Page: https://duck.co/ia/view/currency

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by adityatandon007** [comment]
@moollaza how can I trigger for the queries like `value of bitcoin` or `price of bitcoin` Some help w'd be very useful

File: t/Currency.t [hunk 1]
```diff
@@ -111,13 +111,19 @@ ddg_spice_test(
...
         call_type => 'include',
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
+    'bitcoin price' => test_spice(
+        '/js/spice/currency/1/xbt/xbt',
+        call_type => 'include',
+        caller => 'DDG::Spice::Currency',
+        is_cached => 0
+    ),
```
Comment (pjhampton) [inline t/Currency.t]: I can't get either of these tests to run in the search bar for some reason. Can you?

Comment (adityatandon007) [inline t/Currency.t]: Its working for me in the duckpan server

screenshot from 2017-04-25 14-47-18

File: lib/DDG/Spice/Currency.pm [hunk 2]
```diff
@@ -33,7 +33,7 @@ my $number_re = number_style_regex();
 my $cardinal_re = join(' |', qw(hundred thousand k million m billion b trillion)).' ';
 my $from_qr = qr/(?<fromSymbol>\p{Currency_Symbol})|(?:(?<from>$currency_qr)s?)/;
 my $amount_qr = qr/(?<amount>$number_re*)\s?(?<cardinal>$cardinal_re)?/;
-my $keyword_qr = qr/(?:\s?(?<currencyKeyword>(?:currency|value))\s?)/i;
+my $keyword_qr = qr/(?:\s?(?<currencyKeyword>(?:currency|value|price|))\s?)/i;
```
Comment (moollaza) [inline lib/DDG/Spice/Currency.pm]: @adityatandon007 there shouldn't be a trailing `|` after `|price`. This dramatically changes what the regex matches on, so please remove that 👍

**Comment by adityatandon007** [comment]
@moollaza It was a typo, was unintentional. Thanks for catching that. Changes done as requested. By the way how to trigger for queries like `value of usd`

**Comment by moollaza** [comment]
> By the way how to trigger for queries like value of usd

@adityatandon007 it looks like you'd need to modify the `$question_qr` to allow for `value of` and `price of` as prefixes.

**Comment by moollaza** [comment]
@adityatandon007 Thanks! This LGTM :+1:

**Comment by bsstoner** [comment]
This was released to production 🚀

**Pull request merged**